docs/logs/special-fields.md

```diff
@@ -1,6 +1,8 @@
 title: Special Logs Fields
 description: Sematext's logging management app looks for special fields in logs, namely host, source, facility, severity, syslog-tag, tags, message, and @timestamp
 
+All [Special Tags](../../special-tags) apply as well.
```

Comment:
Maybe reword to something like "Common Special Tags apply to Sematext Logs. Additionally, there are Logs-specific fields which also get special treatment:.."